src/main/java/janet/Ui.java

```diff
@@ -0,0 +1,97 @@
+    static final String EXPECTED_DELETED_INDEX_GOT_NONE = "You need to indicate which task number should be deleted.";
+    static final String EXPECTED_DELETED_INDEX_GOT_OTHER = "The task to be deleted should be indicated its list index.";
+    static final String UNRECOGNISED_OPERATION = "Sorry, I do not understand this command.";
+    static final String INDEX_OUT_OF_BOUNDS_STRING = "Sorry, there is no task with the index number ";
+
+    /**
+     * Returns a string representation of the tasks in the list.
+     *
+     * @param taskList Janet's list of tasks
+     * @return String representation of list
+     */
+    public static String taskListString(TaskList taskList) {
+        assert(taskList != null);
+        if (taskList.size() == 0) {
+            return "I did not find any tasks.";
+        }
+        String output = "Here's what I found:\n";
+        for (int i = 0; i < taskList.size(); i++) {
+            output += String.format("%d. %s\n", i + 1, taskList.get(i));
+        }
+        return output;
+    }
+
+    /**
+     * Returns a string to confirm that a task has been added to the list.
+     *
+     * @param task Task that has been added
+     * @param size Total number of tasks in the list
+     * @return String confirming that the task has been added
+     */
+    public static String addedString(Task task, int size) {
+        assert(task != null);
+        return String.format("Alright, I've added this task: \n"
+                        + "\t%s\n"
+                        + "Now, you have %d tasks in the list.\n",
+                task, size);
+    }
+
+    /**
+     * Returns a string to confirm that a task has marked as complete.
+     *
+     * @param task The task that has been marked as complete
+     * @return String confirming that the task has been marked as complete
+     */
+    public static String doneString(Task task) {
+        assert(task != null);
+        return String.format("Certainly, I've marked this task as done: \n"
+                        + "\t%s\n",
+                task);
+    }
+
+    /**
+     * Returns a string to confirm that a task has been deleted from the list.
+     *
+     * @param task Task that has been deleted
+     * @param size Total number of tasks left in the list
+     * @return String confirming that the task has been deleted
+     */
+    public static String deletedString(Task task, int size) {
+        assert(task != null);
+        return String.format("Certainly, I've deleted this task: \n"
+                        + "\t%s"
+                        + "\nNow, you have %d tasks in the list.\n",
+                task, size);
+    }
+
+    /**
+     * Returns a string that is shown when a user tries to access a non-existent task index.
+     * @param index The index of the non-existent task that the user tried to access
+     * @return String to inform the user of index out of bounds error
+     */
+    public static String outOfBoundsString(int index) {
```

Comment:
Would printOutOfBoundsString be a bit more intuitive here?

Reply:
I'm a little unsure on this, because this method returns a String whereas printXXX might give the impression of a void method that has the side-effect of logging to console. Thanks for pointing it out, I may open an issue to ask for more opinions.